polkadot/node/subsystem-bench/src/lib/environment.rs

```diff
@@ -118,6 +118,7 @@ fn new_runtime() -> tokio::runtime::Runtime {
 		.thread_name("subsystem-bench")
 		.enable_all()
 		.thread_stack_size(3 * 1024 * 1024)
+		.worker_threads(4)
```

Comment:
We need to ensure the number of worker threads chosen here matches:
- the number of CPU cores reserved by K8s/CI for the pod running the bench
- the number of blocking threads used by subsystems to do heavy work